cabal.project

```diff
@@ -15,6 +15,16 @@ index-state:
   , hackage.haskell.org 2025-02-20T07:20:13Z
   , cardano-haskell-packages 2025-02-11T21:18:23Z
 
+source-repository-package
+  type: git
+  location: https://github.com/cardano-scaling/blockfrost-haskell.git
+  tag: 042179583f439d09c271b11bac9e9289133c0a66
+  --sha256: j2kRxDxtehHB0mQ4lHE6PqjOVca2dHgQgJaxtCPDeM8=
+  subdir:
+    blockfrost-client-core
+    blockfrost-api
+    blockfrost-client
```

Comment:
Must: not use a fork just for selecting instances

Reply:
Do you have any ideas on how to do this without removing the instance in the fork?

Reply:
I've tried all I could think about and still couldn't get ghc to pick the instance I want.